pset-functions.js

```diff
@@ -75,6 +111,64 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+function scrabbleScore(word){
+  const scoreTable = scrabbleTable()
+  // capitalize all letters
+  word = word.toUpperCase()
+  letters = word.split("")
+
+  let score = 0
+
+  letters.forEach((letter) => {
+    if(scoreTable[letter] !== undefined){
+      score = score + scoreTable[letter]
+    }
+  });
+  return score
+}
```

Comment:
Katie, lots of great things happening here; the forEach iterating through each element and taking advantage of an object's bracket notation to read the value from the key/value pairing is really well done! In line 130, you are essentially using a function to build an object with the appropriate key/value pair; instead of creating the object outright your function allows you to change points for each group instead of doing individually (although the function setup requires a lot of code). As an alternative solution (purely for your benefit), an example of the method .includes on an array and if/else statements found [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L98).